src/threading/thread.cpp

```diff
@@ -311,7 +311,7 @@ bool Thread::bindToProcessor(unsigned int proc_number)
 
 bool Thread::setPriority(int prio)
 {
-#if USE_WIN_THREADS
+#ifdef _WIN32
```

Comment:
Should be done now. 
the remaining failure seems to be a failure of travis itself